src/components/AddAdvert/AddAdvertForm/AddAdvertForm.jsx

```diff
@@ -0,0 +1,226 @@
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import { Section } from './AddAdvertForm.styled';
+import { FormTitle, Button } from '../../common';
+
+
+const initialValues = {
+    title: '',
+    category: '',
+    subcategory: '',
+    time: '',
+    format: '',
+    price: '',
+    language: '',
+    text: '',
+};
+
+
+const AddAdvertForm = () => {
+    const handleSubmit = (values, { resetForm }) => {
+        console.log(values);
+        resetForm();
+    };
+
+
+    return (
+        <Section>
+            <Formik
+                initialValues={initialValues}
+                onSubmit={handleSubmit}
+            >
+                {({
+                    errors,
+                    touched,
+                    values,
+                    handleChange,
+                    handleBlur,
+                    isSubmitting,
+                }) => (
+                    
+                    <div>
+                        <FormTitle>Create advertisement</FormTitle>
+                        <Form>
+                            {/* // TITLE */}
```

Comment:
The code is clear enough, this comment is redundant

Reply:
has removed